<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageOutputInfo</class>
 <widget class="QWidget" name="ImageOutputInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="imageTypeLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Image Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="imageType">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>png</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bmp</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ppm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tif</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tiff</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>jpg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>jpeg</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2" rowspan="5">
    <widget class="QLabel" name="thumbnailLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="baseSize">
      <size>
       <width>147</width>
       <height>145</height>
      </size>
     </property>
     <property name="text">
      <string>View not currently available</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="imageFileNameLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>File Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="imageFileName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>The file name must contain %t.  This gets replaced with the time step.</string>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="text">
      <string>image_%t.png</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="imageWriteFrequencyLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Write Frequency</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="imageWriteFrequency">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="imageMagnificationLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Magnification</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="imageMagnification">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="fitToScreen">
     <property name="toolTip">
      <string>Check to reset the view to fit the objects to the view every outputted time step.</string>
     </property>
     <property name="whatsThis">
      <string>Check to reset the view to fit the objects to the view every outputted time step.</string>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Fit to Screen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
